lua/lazyvim/plugins/extras/lang/shell.lua

```diff
@@ -0,0 +1,28 @@
+return {
+  {
+    "neovim/nvim-lspconfig",
+    opts = {
+      servers = {
+        bashls = {},
+      },
+    },
+  },
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if type(opts.ensure_installed) == "table" then
+        vim.list_extend(opts.ensure_installed, { "bash" })
+      end
+    end,
+  },
+  {
+    "williamboman/mason.nvim",
+    opts = function(_, opts)
+      vim.list_extend(opts.ensure_installed or {}, {
+        -- formatters & linters
+        "shfmt",
```

Comment:
shfmt is not needed with bashls

Reply:
Scrap that. It **is** needed and I'm actually already using that for formatting :)

Reply:
Moved the changes to dot file instead. I think I also need to change the PR title